hikari/impl/rest.py

```diff
@@ -563,7 +549,7 @@ async def _request(
 
         headers.put(_X_AUDIT_LOG_REASON_HEADER, reason)
 
-        url = compiled_route.create_url(self._rest_url)
+        url = compiled_route.create_url(urls.REST_API_URL)
```

Comment:
constantly accessing the top level constant instead of saving it to the instance and accessing what's saved to the instance has different implications about how top level patching of said constant (which apparently we encourage now?) would effect the behaviour of already active clients